src/results/site.jl

```diff
@@ -47,6 +47,7 @@ function add_site_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
 	r["annual_onsite_renewable_electricity_kwh"] = round(value(m[:AnnualOnsiteREEleckWh]), digits=2)
 	r["onsite_renewable_electricity_fraction_of_elec_load"] = round(value(m[:AnnualOnsiteREEleckWh])/value(m[:AnnualEleckWh]), digits=4)
 	r["onsite_and_grid_renewable_electricity_fraction_of_elec_load"] = round((value(m[:AnnualOnsiteREEleckWh]) + value(m[:AnnualGridREEleckWh])) /value(m[:AnnualEleckWh]), digits=4)
+	r["electric_load_converted_from_thermal_kwh"] = round(value(m[:AnnualEleckWh]) - (sum(p.s.electric_load.loads_kw) * p.hours_per_time_step), digits=2)
```

Comment:
for the naming of this, just confirming that there aren't instances in which REopt competes one electric tech against another more efficient tech, which could change the load? (not sure how important that would even be here). Would you mind adding this to the help text at the top with a brief commented description?

Reply:
I think it should keep the name and include the conversion of cooling loads.  Updated this in https://github.com/NREL/REopt.jl/pull/456/commits/8bd8c937789aa91aaf86e91b31b947e6a6bee130